src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -164,19 +168,16 @@ static ucs_status_t uct_ib_mlx5_devx_reg_ksm_data_addr(
     }
 
     status = uct_ib_mlx5_devx_reg_ksm(md, atomic, iova, length, list_size,
-                                      UCT_IB_MD_MAX_MR_SIZE, in, reason, mr_p,
-                                      mkey);
+                                      UCT_IB_MD_MAX_MR_SIZE, in, index_first,
```

Comment:
why in some places it's mkey_first/last and others it's index_first/last?

Reply:
fixed, using `mkey_index`